src/services/index.ts

```diff
@@ -19,6 +19,7 @@ const publicApiServices = [
   'search-facets',
   'entities',
   'impresso-ner',
+  'subscriptions',
```

Comment:
If it's a public API service - it needs a schema, otherwise it will be hanging in Swagger misleading users.

Reply:
I'll move to internal api only